xwiki-rendering-macros/xwiki-rendering-macro-box/src/test/resources/macrobox8.test

```diff
@@ -10,9 +10,9 @@
 beginDocument
 beginMacroMarkerStandalone [custombox] [] [something]
 beginGroup [[class]=[box]]
-beginMetaData [[non-generated-content]=[java.util.List<org.xwiki.rendering.block.Block>][parameter-name]=[title]]
```

Comment:
Does it mean that the title is not inplace editable anymore? AFAIR those metadata blocks were there on purpose to allow direct edition.

Reply:
Indeed, without this block, it's not possible to select the title for direct inplace editing. We can only change it through the message box properties.

Reply:
:+1: Addressed in https://github.com/xwiki/xwiki-rendering/pull/286/commits/dcd57c288fa3f4f0c25f733fd614bde88ee0e82a

Reply:
Shouldn't the title be a block element like `<div>`? I remember that we had a problem that inline elements weren't editable in CKEditor.

Reply:
AFAICS this works properly with [the changes proposed on xwiki-platform](https://github.com/xwiki/xwiki-platform/pull/2590):

[Demo video](https://up1.xwikisas.com/#tAgXpwwH8OU17qh3nkjTTQ)